packages/params/test/e2e/overridePresetOk.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint import/order: "off" */
+// This script is should be run in an e2e !!
+// It demostrates how to properly change the Lodestar preset safely
+
+// 1. Import from @lodestar/params/setPreset only
+import {setCustomPreset, PresetName} from "../../src/setPreset.js";
+// eslint-disable-next-line @typescript-eslint/naming-convention
+setCustomPreset(PresetName.minimal, {SLOTS_PER_EPOCH: 2});
+
+// 2. Import from any other @lodestar/params paths
+import {expect} from "chai";
```

Comment:
Just use node:assert equals to keep the scripts simple https://nodejs.org/api/assert.html#assertequalactual-expected-message